crates/polars-core/src/config.rs

```diff
@@ -33,6 +33,13 @@ pub fn verbose() -> bool {
     std::env::var("POLARS_VERBOSE").as_deref().unwrap_or("") == "1"
 }
 
+pub fn use_gpu_engine() -> bool {
+    std::env::var("POLARS_DEFAULT_ENGINE")
+        .as_deref()
+        .unwrap_or("")
+        == "gpu"
+}
```

Comment:
Yeah, I think we should just return (and encode the default) engine here? WDYT?

Reply:
Yeah, I agree.